### ISSUE TITLE
Schematics (I know it's testing) won't load any types

### ISSUE THREAD

**Issue by hobolyra** [Zergatul/cheatutils#22]
Loading .schematic files can't be done, nor .schem
![image](https://github.com/Zergatul/cheatutils/assets/112722819/ef26456f-7d82-4b39-a063-f74ad51fa3b7)

**Comment by Zergatul** [comment]
Very strange error, not sure why it happens to you. I just tested, I am able to upload schematic file from forge and fabric versions.
Maybe some other mod uses older version of gson library can cause this?
BTW, it supports also litematic files  (I forgot to change description). For schematic file it converts all blocks to obsidian.

**Comment by hobolyra** [comment]
> 

Yeah not sure why, any attempts to load any format give me the same error. Everything else in the mod works fine, Forge version for 1.19.2 , perhaps that version is the issue since it's not 1.19.4?

**Comment by Zergatul** [comment]
It was never tested in 1.19.2. You get the same error because of serialization code in google json library, not in actual schematics code.
I will only update 1.19.4 going further.

**Comment by hobolyra** [comment]
Makes sense, thanks again for this tool!
